docs/tutorials/tutorial2.md

```diff
@@ -14,148 +14,53 @@ As reported in our paper, we trained an AtacWorks model to learn a mapping from
+In order to vary output file names or formats, or inference parameters, you can change the arguments supplied in `infer_config.yaml`. Type `python AtacWorks/scripts/main.py infer --help` to understand which arguments to change.
 
+In particular, the threshold for peak calling is controlled by the `infer_threshold` parameter in `infer_config.yaml`. By default, this is set to 0.5. If `infer_threshold` is set to "None" in the config file, `run_inference.sh` will instead produce a bigWig file in which each base is labeled with the probability (between 0 and 1) that it is part of a peak. 
 
-## Appendix 1: Output the peak probabilities in inference instead of peak calls
+## Tutorial Notebook
 
-The model predicts the probability of every position on the genome being part of a peak. In the above command, we take a cutoff of 0.5, and output the positions of regions where the probability is greater than 0.5. To output the probability for every base in the genome without any cutoff, we use the following command:
-```
-python $atacworks/main.py infer \
-    --files NK.50_cells.h5 \
-    --sizes_file $atacworks/data/reference/hg19.auto.sizes \
-    --config configs/infer_config.yaml \
-    --config_mparams configs/model_structure.yaml
-    --infer_threshold None
-```
-The inference results will be saved in the folder `output_latest`. This folder will contain the same 4 files described in Step 7. However, `NK_inferred.peaks.bedGraph` and `NK_inferred.peaks.bw` will contain the probability of being part of a peak, for every position in the genome. This command is significantly slower, and the `NK_inferred.peaks.bedGraph` file produced by this command is larger than the file produced in Step 7.
-
-The above command is useful in the following situations:
-1. To calculate AUPRC or AUROC metrics.
-2. If you are not sure what probability threshold to use for peak calling and want to try multiple thresholds.
-3. If you wish to use the MACS2 subcommand `macs2 bdgpeakcall` for peak calling.
-
-To call peaks from the probability track generated by this command, you can use `macs2 callpeak` from MACS2 (link) with the following command:
-```
-macs2 bdgpeakcall -i output_latest/inferred.peaks.bedGraph -o output_latest/inferred.peaks.narrowPeak -c 0.5
-```
-Where `0.5` is the probability threshold to call peaks. Note that the summit calls and peak sizes generated by this procedure will be slightly different from those produced by steps 7-8.
+The above one step command is a good place to get started. For advanced usages, this command is not suitable. For easy demonstration of inference workflow, we've setup a notebook. This notebook breaks the above command down into it's component parts and familiarises the users with each step. Follow the [notebook](https://github.com/clara-parabricks/AtacWorks/blob/dev-v0.3.0/tutorials/tutorial2.ipynb) for step-by-step guide.  
```

Comment:
Minor edits:
"The above one step command is a good place to get started. For advanced usages, this command is not suitable. For easy demonstration of the inference workflow, we've set up a notebook. This notebook breaks the above command down into it's component parts and familiarizes users with each step and all of the options available. Follow the [notebook](https://github.com/clara-parabricks/AtacWorks/blob/dev-v0.3.0/tutorials/tutorial2.ipynb) for this step-by-step guide to denoising and peak calling using a trained AtacWorks model. "

Reply:
done